python/daqconf/apps/trigger_gen.py

```diff
@@ -245,16 +245,16 @@ def get_trigger_app(
                           plugin = 'TCBuffer',
                           conf = get_buffer_conf(TC_SOURCE_ID["source_id"], DATA_REQUEST_TIMEOUT))]
     if USE_HSI_INPUT:
-        modules += [DAQModule(name = 'tctee_ttcm',
+        modules += [DAQModule(name = 'tctee_t',
```

Comment:
This naming seems inconsistent (fake hsi = tcmfake but real hsi (timing) = t?); but it is propagated correctly (just checking you wanted the names like this).